_posts/2017-08-25-introducing-sandboxfs.md

```diff
@@ -3,40 +3,46 @@ layout: posts
 title: Introducing sandboxfs
 ---
 
+sandboxfs is a new sandboxing technique for Bazel. It's experimental and
+subject to change, but it's available now for you to check out!
+
+# Correct builds
+
 As our motto claims, *correctness* is an integral part of Bazel. To achieve
 correctness, builds must be *hermetic* and
 [*reproducible*](https://reproducible-builds.org/), which means that all
 actions invoked by Bazel should be run in a well-defined environment. In other
 words: we want actions to run within a sandbox.
 
-To better illustrate this point, let's consider an example: think of a C++
-`cc_library` target that specifies `foo.cc` as a source. This target contains a
+Let's consider an example. Think of a `cc_library` target that specifies
```

Comment:
I've gone through to word carefully around "target" and "rule".

Reply:
Are you sure this is correct? It's true that, in general, `cc_library` is a rule, but in this specific case I'm talking about a specific target definition (because `foo.cc` is specific to the target, not the rule).

I've changed "The blah rule" to "This blah target", which I feel addresses your concern?

Reply:
I understand. I was trying to be specific about the fact that the action is defined by the rule, but you're right that what this action is doing is specific to the target. (an aside, but - could a rule have actions that aren't used by a specific target, depending on the target's attributes?)

The following is wordier, but is it correct? I'm okay with leaving your current wording if you prefer.

"This cc_library rule contains a compilation action that for this target turns the foo.cc source file into a foo.o object file."

Reply:
Yeah, that'd be good too. Done.